Portfolio2.css

```diff
@@ -0,0 +1,222 @@
+* {
+	margin:0;
+	padding: 0;
+	
+}
+
+body {
+	background-image: url("https://wallpaperaccess.com/full/1128684.jpg");
+	background-size: cover;
+	
+	
+ }
+
+header {
+	display: inline-flex; 
+	position: fixed; 
+	margin-left: 1200px;
+	position:fixed;
+	margin-top:-180px;
+}
+
+.menue ul li {
+	display: inline-flex;
+	list-style: none;
+	cursor:pointer;
+	padding:25px;
+	
+}
+
+.logo img {
+	width: 90px;
+	margin-top: -7px;
+	margin-right: 48px;
+	
+}
+
+.active {
+	color: grey
+}
+.activ {
+	color: grey;
+}
+
+.acti {
+	color: grey;
+}
+
+#headertwo {
+        background-color: none;
+    	width: 100%;
+    	text-align: right;
+	font-size: 1cm;
+	margin-top: 200px;
+	font-family: garamond;
+  	color: #7c7b85;
+	padding: 14px 16px;
+  	text-decoration: none;
+	margin-left: 100px ;
```

Comment:
Don't use margin like this to position elements. You've centred your banners by giving them a 90% width and margin: 0 auto; which is the perfect way of doing it.

Do the same thing here, remember my jumbotron example build? Have a look at how the containers work in that to see an example.